av_node:
  ros__parameters:
    publishers:
      raw_frame: /arm_camera/raw_frame

    services:
      stop_camera: /arm_camera/get_cam_state

    frames: 
      camera_link: wrist_camera_link
